pulseapi/entries/serializers.py

```diff
@@ -89,6 +89,19 @@ def get_published_by(self, instance):
         """
         return instance.published_by.name
 
+    # "virtual" property so that we can link to the correct profile
+    submitter_profile_id = serializers.SerializerMethodField()
+
+    def get_submitter_profile_id(self, instance):
+        """
+        Get the id for the user who published this entry
+        """
+        profiles = instance.published_by.profile.all()
```

Comment:
this is a little odd, but a consequence of how we ended up implementing profiles. They're technically a many-to-many relation (and I already can't remember why) so accessing `instance.published_by.profile` yields a RelationManager, not an actual UserProfile object.

Reply:
it looks like it's a one-to-many relation, i.e. 1 User has many profiles. Yeah weird, it probably should have been 1:1 (optional relationship due to orphaned org profiles).